app/workers/process_photo.rb

```diff
@@ -15,6 +15,15 @@ def perform(id)
 
       photo.processed_image.store!(unprocessed_image)
 
+      ##unless current_user.strip_exif
+		  exifdata = MiniExiftool.new unprocessed_image.path, :coord_format => '%+.18f'
+
+		  photo.location =
+		  if exifdata.GPSLatitude and exifdata.GPSLongitude
```

Comment:
- Tab detected.
- Indent the first parameter one step more than the start of the previous line.
- Indent the first line of the right-hand-side of a multi-line assignment.
- Use `&&` instead of `and`.